packages/ml-vision/ios/RNFBMLVision/RNFBMLVisionTextRecognizerModule.m

```diff
@@ -0,0 +1,192 @@
+        [RNFBSharedUtils rejectPromiseWithUserInfo:reject userInfo:(NSMutableDictionary *) @{
+            @"code": @"unknown",
+            @"message": [error localizedDescription],
+        }];
+        return;
+      }
+
+      resolve([self getFirebaseVisionTextMap:text]);
+    }];
+  }];
+}
+
+- (NSDictionary *)getFirebaseVisionTextMap:(FIRVisionText *)text {
+  NSMutableDictionary *firebaseVisionTextMap = [[NSMutableDictionary alloc] init];
+
+  firebaseVisionTextMap[@"text"] = text.text;
+  firebaseVisionTextMap[@"blocks"] = [self getVisionTextBlocksList:text.blocks];
+
+  return firebaseVisionTextMap;
+}
+
+- (NSArray *)getVisionTextBlocksList:(NSArray <FIRVisionTextBlock *> *)blocks {
+  NSMutableArray *blockListFormatted = [[NSMutableArray alloc] init];
+
+  for (FIRVisionTextBlock *block in blocks) {
+    NSMutableDictionary *textBlockFormatted = [[NSMutableDictionary alloc] init];
+
+    textBlockFormatted[@"text"] = block.text;
+    textBlockFormatted[@"confidence"] = block.confidence;
+    textBlockFormatted[@"recognizedLanguages"] = [self getLanguageCodesList:block.recognizedLanguages];
+//    textBlockFormatted[@"cornerPoints"] = [RNFBMLVisionCommon visionPointsToArray:block.cornerPoints];
+    textBlockFormatted[@"cornerPoints"] = @[];
+    textBlockFormatted[@"lines"] = [self getLinesList:block.lines];
+
+    [blockListFormatted addObject:textBlockFormatted];
+  }
+
+  return blockListFormatted;
+}
+
+- (NSArray *)getLinesList:(NSArray <FIRVisionTextLine *> *)lines {
+  NSMutableArray *lineListFormatted = [[NSMutableArray alloc] init];
+
+  for (FIRVisionTextLine *line in lines) {
+    NSMutableDictionary *lineFormatted = [[NSMutableDictionary alloc] init];
+
+    lineFormatted[@"boundingBox"] = [RNFBMLVisionCommon rectToIntArray:line.frame];
+    lineFormatted[@"text"] = line.text;
+    lineFormatted[@"confidence"] = line.confidence;
+    lineFormatted[@"recognizedLanguages"] = [self getLanguageCodesList:line.recognizedLanguages];
+//    lineFormatted[@"cornerPoints"] = [RNFBMLVisionCommon visionPointsToArray:line.cornerPoints];
+    lineFormatted[@"cornerPoints"] = @[];
+    lineFormatted[@"elements"] = [self getElementsList:line.elements];
+
+    [lineListFormatted addObject:lineFormatted];
+  }
+
+  return lineListFormatted;
+}
+
+- (NSArray *)getElementsList:(NSArray <FIRVisionTextElement *> *)elements {
+  NSMutableArray *elementsListFormatted = [[NSMutableArray alloc] init];
+
+  for (FIRVisionTextElement *element in elements) {
+    NSMutableDictionary *elementFormatted = [[NSMutableDictionary alloc] init];
+
+    elementFormatted[@"boundingBox"] = [RNFBMLVisionCommon rectToIntArray:element.frame];
+    elementFormatted[@"text"] = element.text;
+    elementFormatted[@"confidence"] = element.confidence;
+    elementFormatted[@"recognizedLanguages"] = [self getLanguageCodesList:element.recognizedLanguages];
+//    elementFormatted[@"cornerPoints"] = [RNFBMLVisionCommon visionPointsToArray:element.cornerPoints];
```

Comment:
Corner points to point array issue

Reply:
Fixed in commit I just pushed from your Mac